internal/notifications/interface.go

```
@@ -0,0 +1,12 @@
+package notifications
+
+import (
+	"context"
+)
+
+var GetNotificationClient func(ctx context.Context) NotificationClient
+
+type NotificationClient interface {
+	SuccessfulLaunch(ctx context.Context, reservationId int64)
+	FailedLaunch(ctx context.Context, reservationId int64, jobError string)
+}
```

Nice, so the API is designed in a way that it ignores errors. It is fine.